lib/styles.js

```diff
@@ -91,7 +91,7 @@ module.exports = function (expect) {
             .jsString("'");
     });
 
-    expect.addStyle('property', function (key, inspectedValue, isArrayLike) {
+    expect.addStyle('propertyForObject', function (key, inspectedValue, isArrayLike) {
```

Comment:
I was actually thinking the same, but wasn't sure what our mechanism for deprecation warnings was? The rename was originally an aid to help tracking down all the places using it, but I think keeping it and warning is great to avoid 'silently' using it.

Reply:
not renaming it would of cause be better :O)